app/notifications/rest.py

```diff
@@ -32,6 +35,150 @@
 register_errors(notifications)
 
 
+@notifications.route('/notifications/email/ses', methods=['POST'])
```

Comment:
Process SES callback. Lots of if statements as we inspect the JSON etc. Probably a bit wordy but as unable to wire up locally thought we may need some info on the preview/staging env.